README.md

```diff
@@ -97,6 +97,26 @@ user in data form there is no advantage to using apache arrow as a query result.
 
 ## Usage
 
+### Preferred method (Nix)
+
+You can get a jar for your own OS (Linux/MacOS) with duckdb included by [installing Nix](https://github.com/DeterminateSystems/nix-installer),
+then running:
+
+``` console
+$ nix build github:schemamap/tmducken
```

Comment:
If you approve the merging of this PR, will update this to reflect the correct org, post-merge.

Reply:
Approved!!